mne_g3d/viz.py

```diff
@@ -0,0 +1,127 @@
+import logging
+
+import ipyvolume as ipv
+from ipyvolume.pylab import style
+import nibabel
+import numpy as np
+
+def read_brain_mesh(surface_path: str):
+    """Function reads triangular format Freesurfer brain surface.
+
+    Parameters
+    ----------
+    surface_path: :obj: `str`
+        Path to the brain surface.
+
+    Returns
+    -------
+    :obj: `numpy.array`
+        Array of vertex (x, y, z) coordinates, of size number_of_vertices x 3.
+    :obj: `numpy.array`
+        Array defining mesh triangles, of size number_of_faces x 3.
+
+    """
+
+    vertices, faces = nibabel.freesurfer.read_geometry(surface_path)
+    faces = faces.astype(np.uint32)
+
+    return vertices, faces
+
+
+def plot_brain_mesh(rh_vertices=None,
+                    lh_vertices=None,
+                    rh_faces=None,
+                    lh_faces=None,
+                    rh_color='grey',
+                    lh_color='grey',
+                    fig_size=(500, 500),
+                    azimuth=90,
+                    elevation=90):
+    """Function for plotting triangular format Freesurfer surface of the brain.
+
+    Parameters
+    ----------
+    rh_vertices : :obj: `numpy.array`, optional
+        Array of right hemisphere vertex (x, y, z) coordinates, of size number_of_vertices x 3.
```

Comment:
Okay it's time to get PEP8 checking working :)

To save you some pain I might configure Travis for you for this repo.

Reply:
Ok, thanks. I have run flake8 check on my machine,but  it hasn't show anything, but maybe it is a matter of additional configurations.

Reply:
```
larsoner@bunk:~/custombuilds/mne-gsoc2018-3d$ flake8 mne_g3d
mne_g3d/viz.py:8:1: E302 expected 2 blank lines, found 1
mne_g3d/viz.py:8:33: E999 SyntaxError: invalid syntax
mne_g3d/viz.py:45:80: E501 line too long (95 > 79 characters)
mne_g3d/viz.py:48:80: E501 line too long (94 > 79 characters)
mne_g3d/viz.py:51:80: E501 line too long (84 > 79 characters)
mne_g3d/viz.py:57:80: E501 line too long (89 > 79 characters)
mne_g3d/viz.py:57:90: W291 trailing whitespace
mne_g3d/viz.py:58:80: E501 line too long (95 > 79 characters)
mne_g3d/viz.py:61:80: E501 line too long (88 > 79 characters)
mne_g3d/viz.py:61:89: W291 trailing whitespace
mne_g3d/viz.py:62:80: E501 line too long (95 > 79 characters)
mne_g3d/viz.py:67:80: E501 line too long (83 > 79 characters)
mne_g3d/viz.py:69:80: E501 line too long (91 > 79 characters)
mne_g3d/viz.py:89:1: W293 blank line contains whitespace
mne_g3d/viz.py:102:80: E501 line too long (87 > 79 characters)
mne_g3d/viz.py:111:80: E501 line too long (97 > 79 characters)
mne_g3d/viz.py:111:106: W291 trailing whitespace
mne_g3d/viz.py:112:80: E501 line too long (88 > 79 characters)
mne_g3d/viz.py:124:1: W293 blank line contains whitespace
```

Reply:
I have fixed styling error and updated docstrings.